ApplicationLibCode/Commands/EclipseCommands/RicImportEclipseCaseFeature.cpp

```diff
@@ -59,9 +59,10 @@ void RicImportEclipseCaseFeature::onActionTriggered( bool isChecked )
     defaultDir = QFileInfo( fileNames.last() ).absolutePath();
     app->setLastUsedDialogDirectory( "BINARY_GRID", defaultDir );
 
-    bool             createDefaultView = false;
-    std::vector<int> caseIds;
-    openEclipseCaseFromFileNames( fileNames, createDefaultView, caseIds );
+    bool                               createDefaultView = false;
+    std::vector<int>                   caseIds;
+    std::shared_ptr<RifReaderSettings> readerSettings;
```

Comment:
Is it correct to use default settings here?

Reply:
Yes, I think so. This is the "Import Eclipse Case" menu item, and I did not want to change the behavior (it still uses values from RiaPreferences).